scene/gui/tree.cpp

```diff
@@ -3233,7 +3233,8 @@ void Tree::_go_up() {
 		if (!prev) {
 			return;
 		}
-		selected_item = prev;
+
+		select_single_item(prev, get_root(), selected_col);
 		emit_signal(SNAME("cell_selected"));
```

Comment:
This will now be emitted twice

Reply:
Removed `emit_signal` in the two places it appeared under `select_single_item`